eventfinder/src/eventcard.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```

Comment:
The EventCard component looks great!

Sometimes you save a reviewer's time by providing screenshots in the description of the pull request. There is already an issue created for later to adjust the PR template, which will remind you about adding that information.

It may seem nit-picky, but there are ways to make your PR more effective. One of them is to keep it short and sweet. The longer a person reviews a PR, the more the quality declines. I've had times when I had to review thousands of lines of code, felt physically fatigued, and had to return later. People often procrastinate those reviews as well.

![image](https://user-images.githubusercontent.com/16828669/89063611-7736db80-d336-11ea-9d8f-703190532247.png)